openwrt/nodogsplash/Makefile

```diff
@@ -7,8 +7,8 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=nodogsplash
 PKG_FIXUP:=autoreconf
-PKG_VERSION:=3.3.3
-PKG_RELEASE:=1-beta
+PKG_VERSION:=4.0.0-beta
```

Comment:
4.0.0-beta?

Reply:
Yes - see previous comment. I have been compiling the package and testing and it is useful to see which test version you are at. Can fix these now if you like.